Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -62,6 +62,7 @@ type DefaultProps = {
   renderLeftComponent: SubViewRenderer,
   renderRightComponent: SubViewRenderer,
   renderTitleComponent: SubViewRenderer,
+  statusBarHeight: number,
```

Comment:
Flow is failing because `statusBarHeight` is `number` in `defaultProps`, but `number | Animated.Value` in `props`.